tests/puppeteer/campaigns/data/demo/customer.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  defaultAccount: {
+  DefaultAccount: {
```

Comment:
Does it change somewhere ?

Reply:
Yes, I have corrected all files affected by my changes ;)